### PR TITLE
[Naitei][Ecommerce]   send mail after order

### PR DESCRIPTION
**Pull request by SuKay712**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/78429

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

https://github.com/user-attachments/assets/7d9032a3-e5dd-4e7d-adb7-e0ca21dde997

### CODE REVIEW

**Comment by SuKay712** [comment]
ready

**Comment by SuKay712** [comment]
ready

File: quit [hunk 1]
```diff
@@ -0,0 +1,13 @@
+=> [32m#<User:0x00007f6b8daf4498[0m
```
Comment (ghost) [inline quit]: file này ignore đi em

File: app/views/bill_mailer/bill_info.html.erb [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Order Confirmation</title>
```
Comment (ghost) [inline app/views/bill_mailer/bill_info.html.erb]: i18n đi em

**Review by ghost — COMMENTED** [review]
cmt

**Comment by SuKay712** [comment]
ready

**Pull request merged**